static/js/dashboard_routes.js

```diff
@@ -0,0 +1,99 @@
+import App from './containers/App';
+
+const errorLoading = error => {
+  throw new Error(`Dynamic page loading failed: ${error}`);
+};
+
+const loadRoute = cb => (
+  module => cb(null, module.default)
+);
+
+const username = SETTINGS.user ? SETTINGS.user.username : '';
+
+export const routes = {
```

Comment:
One solution that would allow us to potentially unify them is to use webpack to run the tests - but I'm not sure how complicated and messy that would be (my thought is probably very messy)